### PR TITLE
Update _form.html.erb to fix #9934

### LINKED ISSUE #9934: Fix profile tag input bar
Hi, this is a [first-timers-only issue](https://code.publiclab.org/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

- [ ] 📝 **Update** the file [_form.html.erb](https://github.com/publiclab/plots2/blob/1afcac6d61715fcd346e3b7148a7505697297ca1/app/views/tag/_form.html.erb#L12) in the `plots2` repository (press the little pen Icon) and edit the line as shown below. 

[See this page](https://code.publiclab.org/#r=all) for some help in taking your first steps!

Below is a "diff" showing in red (and a `-`) which lines to remove, and in green (and a `+`) which lines to add:

```diff
-        <input aria-label="Enter tag name" autocomplete="off" class="tag-input form-control" name="name" type="text" placeholder="<%= translation('tag._tagging.enter_tags') %>" data-provide="typeahead" />
+        <input aria-label="Enter tag name" autocomplete="off" class="tag-input form-control" name="name" type="text" placeholder="<%= translation('tag._tagging.enter_tags,{},false') %>" data-provide="typeahead" />
```

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 
- [ ] As this issue requires UI checks please add screenshots of changes in PR comment
  - [ ]  Run the changes in the Gitpod - option to do so will appear in your PR once you create a PR 
  - [ ] You will find an option to switch on Translation
  - [ ] Attach a screenshot/gif of the profile page showing input bar with Translation on and off  
This is a sample screenshot, you can add a similar in your PR
Screenshot from 2021-07-21 16-08-48

- [ ] 🏁 **Done** Ask in comments for a review :)

### Please keep us updated

💬⏰ - We encourage contributors to be respectful to the community and provide an update **within a week** of claiming a first-timers-only issue. We're happy to keep it assigned to you as long as you need if you update us with a request for more time or help, but if we don't see any activity a week after you claim it we may reassign it to give someone else a chance. Thank you in advance! 

If this happens to you, don't sweat it! Grab another open issue.

### Is someone else already working on this?

🔗- We encourage contributors to link to the original issue in their pull request so all users can easily see if someone's already started on it. 

👥- **If someone seems stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://code.publiclab.org/#r=all). Thanks!

### 🤔❓ Questions?

Leave a comment below!

### PR DESCRIPTION
**Pull request by Pandademic**
Fixes: #9934
Issue:[#9934 ](https://github.com/publiclab/plots2/issues/9934)

### CODE REVIEW

**Comment by Pandademic** [comment]
@publiclab/reviewers I need help

File: mychanges/img/Readme [hunk 1]
```diff
@@ -0,0 +1 @@
+Screenshots for my changes are stored here
```
Comment (imajit) [inline mychanges/img/Readme:1]: Are you trying to share screenshots of the changes? if so you can add them in the comments of your PR instead

**Comment by imajit** [comment]
Hello @Pandademic , Welcome to Public Lab, and congrats on your first PR :rocket: :rocket: :rocket: 
I see some of the tests are failing, maybe you can retry by changing the form file only and adding the screenshots as comments.
Feel free to tag me or any other reviewers if you are still stuck.

File: mychanges/img/Readme (hunk 1 above)
Comment (Pandademic) [reply mychanges/img/Readme:1]: Ok,thanks!

Comment (Pandademic) [reply mychanges/img/Readme:1]: I am actually having trouble running the files to take the screenshots

**Comment by Pandademic** [comment]
Ok thanks

**Comment by Pandademic** [comment]
Hi @imajit,Should I pull again with only the modified file and the images as comments?

**Comment by imajit** [comment]
You can commit new changes to this PR without pulling.

File: mychanges/img/Readme (hunk 1 above)
Comment (imajit) [reply mychanges/img/Readme:1]: You can run your changes in Gitpod, it is similar to local environment

**Comment by Pandademic** [comment]
Ok great ,thanks!

**Comment by Pandademic** [comment]
I am really sorry if this sounds stupid but how to I run this in gitpod?  😅

**Comment by imajit** [comment]
You can click [here](https://github.com/publiclab/plots2/pull/9938#issuecomment-884493342) and once the code is loaded, you will see an option on the bottom rigth corner to open browser view.

**Review by Tlazypanda — APPROVED** [review]
Hey @Pandademic Great work :tada: looks like all the tests are passing now

**Comment by Pandademic** [comment]
Thank you @Tlazypanda

**Comment by Pandademic** [comment]
Ok thanks @imajit

**Comment by noi5e** [comment]
@Pandademic This looks great!!! Thanks so much for your first contribution, and welcome to Public Lab! Merging now.

**Comment by Pandademic** [comment]
Thank you!

**Pull request merged**